src/test/data/JsonSerializableStudentListTest/duplicateStudentStudentList.json

```diff
@@ -4,7 +4,7 @@
     "course": "CS2103T",
     "email": "alice@example.com",
     "remark" : "",
-    "tags": [ "friends" ]
+    "tags": [ "GOOD" ]
```

Comment:
Is there any particular reason for using "tags" plural instead of "tag" singular? I noticed the same usage in other tests as well